src/types/gecko-profile.js

```diff
@@ -237,8 +237,12 @@ export type GeckoThread = {|
   'eTLD+1'?: string,
   // If present and true, this thread was launched for a private browsing
   // session only.
+  // It's absent in Firefox 97 and before, or in Firefox 98+ when this thread
+  // had no extra attribute at all.
```

Comment:
What does `when this thread had no extra attribute at all.` mean exactly? You mean when it's not a private browsing, or when we don't add any attribute for some reason? I see the same comment in a few places in this commit as well.

Reply:
They represent the "origin attributes" as implemented in https://searchfox.org/mozilla-central/source/caps/OriginAttributes.cpp. They're encapsulated in the origin information on thread object, this is the patch where I get them: https://phabricator.services.mozilla.com/D129818

I will try to make the comment more precise.
I believe that in non-fission they're always missing, but that this is also the case in "normal" threads in Fission. (but not 100% sure)